modular_skyrat/modules/colony_fabricator/code/machines/rtg.dm

```diff
@@ -0,0 +1,38 @@
+/obj/machinery/power/rtg
```

Comment:
Wait were old RTGs removed entirely and I missed it, or do they not use this path? 
... or, are you just overwriting their base version to also be portable? Either way it's cool just wanted some clarity. If it's the latter make sure the base RTG isn't used anywhere you don't want it irradiating